dev/integration_tests/codegen/pubspec.yaml

```diff
@@ -0,0 +1,28 @@
+name: codegen
+description: A test of Flutter integrating code generation.
+
+environment:
+  # The pub client defaults to an <2.0.0 sdk constraint which we need to explicitly overwrite.
+  sdk: ">=2.0.0-dev.68.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_driver:
+    sdk: flutter
+  flutter_build:
+    sdk: flutter
+  inject:
```

Comment:
It's a little strange that this didn't bring any other dependencies, such as `code_builder` and `dart_style`.

Reply:
I have not run update-packages yet, since that makes the diff much larger